models/user.js

```diff
@@ -43,10 +51,15 @@ const joiLoginSchema = Joi.object({
   email: Joi.string().required(),
 });
 
+const verifyEmailSchema = Joi.object({
+  email: Joi.string().required(),
```

Comment:
To trzeba bardziej zacieśnić, Joi sprawdza też wiadomości email.

Reply:
Czyli np dodać email(). ?